airflow-core/docs/core-concepts/executor/index.rst

```diff
@@ -239,7 +239,8 @@ Important BaseExecutor Methods
 These methods don't require overriding to implement your own executor, but are useful to be aware of:
 
 * ``heartbeat``: The Airflow scheduler Job loop will periodically call heartbeat on the executor. This is one of the main points of interaction between the Airflow scheduler and the executor. This method updates some metrics, triggers newly queued tasks to execute and updates state of running/completed tasks.
-* ``queue_command``: The Airflow Executor will call this method of the BaseExecutor to provide tasks to be run by the executor. The BaseExecutor simply adds the TaskInstances to an internal list of queued tasks within the executor.
+* ``queue_command``: Airflow 2 way of doing things. The Airflow Executor will call this method of the BaseExecutor to provide tasks to be run by the executor. The BaseExecutor simply adds the TaskInstances to an internal list of queued tasks within the executor. CeleryK8s and LocalK8s executors are examples of this.
+* ``queue_workload``: Airflow 3 way of doing things. The Airflow Executor will call this method of the BaseExecutor to provide tasks to be run by the executor. The BaseExecutor simply adds the workloads to an internal list of queued workloads to run within the executor. All in-tree executors except the ones mentioned above are using this.
```

Comment:
Many users won't really know what 'in-tree' means. And I'm not sure it's too helpful listing them here.

If anything I'd create a new section to list executors that support task SDK, and we can have them clearly documented there, rather than buried deep in this technical text.

Reply:
Yeah that is removed now